platforms/core-execution/workers/src/main/java/org/gradle/workers/internal/DefaultWorkerExecutor.java

```diff
@@ -72,17 +72,20 @@ public class DefaultWorkerExecutor implements WorkerExecutor {
     private final Instantiator instantiator;
     private final IsolationScheme<WorkAction<?>, WorkParameters> isolationScheme = new IsolationScheme<>(Cast.uncheckedCast(WorkAction.class), WorkParameters.class, WorkParameters.None.class);
     private final CachedClasspathTransformer classpathTransformer;
-    private final File baseDir;
+    private final File projectDir;
```

Comment:
❌ Let's not rename this. For some workers it's not the project directory but the "fork options" working directory. `baseDir` is fine.

Reply:
Will do so, nothing against reverting it, however I can't find any case in the code when it's not the project directory. Just saying.